webapp-django/crashstats/crashstats/jinja2/crashstats/signup.html

```diff
@@ -10,34 +10,32 @@ <h2>Signing up for an account on Crash Stats</h2>
 
     <div class="panel">
       <div class="body">
-        <div class="prose">
```

Comment:
I got rid of "prose" as a class and switched to using tailwind-inspired classes.